app/src/main/java/com/github/se/icebreakrr/ui/sections/shared/DisplayProfile.kt

```diff
@@ -107,13 +115,8 @@ fun InfoSection(profile: Profile, tagsViewModel: TagsViewModel) {
             verticalArrangement = Arrangement.spacedBy(VERTICAL_ARRANGEMENT)) {
               Text(
                   text = "Description",
-                  style =
-                      TextStyle(
-                          fontSize = 14.sp,
-                          lineHeight = 20.sp,
-                          fontWeight = FontWeight.W500,
-                          color = MaterialTheme.colorScheme.secondary,
-                          letterSpacing = 0.1.sp))
+                  style = INFO_SUBTITLE_STYLE,
```

Comment:
Wow I didn't know that you can use styling in values like that, nice to see!

Reply:
Yes ! I wished we had more time so we could have centralized all text styles in a single class for a more consiistent UI